packages/astro/src/core/errors/dev/vite.ts

```diff
@@ -150,9 +150,15 @@ export async function getViteErrorPayload(err: ErrorWithMetadata): Promise<Astro
 		: undefined;
 
 	const highlighter = await getHighlighter({ theme: 'css-variables' });
+	let highlighterLang = err.loc?.file?.split('.').pop();
+	if (['cjs', 'mjs'].includes(highlighterLang ?? '')) {
```

Comment:
nit: you could move this array in a `const` at the top of the file, so it's not recreated every time